client/javascript/views/addpatient.ejs

```diff
@@ -16,59 +16,84 @@
     }
   </style>
 </head>
-
-<body class="container">
-  <h1>Add a Patient or Donor</h1>
+<header class="" style="padding-bottom: 20px; padding-top: 10px;">
+  <nav class="navbar navbar-expand-lg navbar navbar-light"  >
+    <a class="navbar-brand" href="/hos" style="padding-left: 25px;"><h2>Indriya</h2></a>
+    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+      <span class="navbar-toggler-icon"></span>
+    </button>
+  
+    <div class="collapse navbar-collapse" id="navbarSupportedContent">
+      <ul class="navbar-nav mr-auto">
+        <li class="nav-item active px-2">
+          <a class="nav-link" href="/hos/registerpatient"><h3>Add a Patient or Doner</h3></a>
+        </li>
+        <li class="nav-item px-2">
+          <a class="nav-link" href="/hos/match"><h3>Find a match</h3></a>
+        </li>
+        
+        <li class="nav-item px-2">
+          <a class="nav-link" href="#"><h3>About</h3></a>
+        </li>
+      </form>
+    </div>
+  </nav>
+</header>
+<body class="container-fluid ">
+  <div style="min-height: 90%;"> 
+  <h1 style="padding-left: 5%;">Add a Patient or Donor</h1>
   <hr>
-  <form class="row g-3" action="/createpatient" method="post">
-
+  <form class="blockquote " action="/createpatient" method="post" style="padding-left: 5%;">
+<div class="row">
     <div class="col-md-2">
       <label class="form-label" for="autoSizingSelect">Type</label>
-      <select class="form-select" name="docType">
+      <select class="form-select" name="docType" required>
         <option value="patient">Patient</option>
         <option value="donor">Donor</option>
       </select>
     </div>
     <div class="col-md-2">
       <label for="name" class="form-label">ID</label>
-      <input type="text" class="form-control" name="PID">
+      <input type="text" class="form-control" name="PID" required>
```

Comment:
Could you add a javascript validation that no special characters are allowed?
client-side javascript to do that -

```
        function alpha(e) {
    var k;
    document.all ? k = e.keyCode : k = e.which;
    return ((k > 64 && k < 91) || (k > 96 && k < 123) || k == 8 || k == 32 || (k >= 48 && k <= 57));
        }
```

Reply:
done